app/views/landing/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render partial: 'header' %>
+<div class="shift-down">
+	<div class="row medium"></div>
+	<div class= "row-break"></div>
+	
+	<% @products.each_with_index do |products, index| %>
+	<div class= "row container-trim-wide"><h5 class="cat-heading"><%= @categories[index].upcase %></h5></div>
+	<div class="container-large">
+		<div class= "row">
+			<%= render partial: "product", collection: products %>
```

Comment:
You're putting every product in one row and separating them with spaces. This won't scale. Lol. There should be a better way